cmd/trigger_ui/index.html

```diff
@@ -9,39 +9,64 @@
 	
 	<form id="requestForm" method="post">
 
-		<h1 for="vxID" id="vxID" name="vxID">{{ .title }}</h1>
+		<h1 for="vxID" id="vxID" name="vxID">{{ .Title }}</h1>
 
 		<div class="checkboxList">
 			<label for="Destinations">Destinations</label>
 			{{range $index, $element := .AssetExportDestinations}}
 			<li>
-				<input type=checkbox name="Destinations[]" value={{$element}}>
+				<input type=checkbox name="destinations[]" value={{$element}}>
 				<label for={{$element}}>{{ $element }}</label>
 				
 			</li>
 			{{end}}
 		</div>
 
-
 		<div class="dropdown">
 			<label for="audioSource">Audio Source</label>
 			<select name="audioSource" id="audioSource" >
+				{{$selectedAudio := .SelectedAudioSource}}
+
 				{{range $index, $element := .AudioSources}}
+					{{if eq $element $selectedAudio}} 
+					<option value={{$element}} selected>{{$element}}</option>
+					{{else}}
 					<option value={{$element}}>{{$element}}</option>
+					{{end}}
 				{{end}}
 			</select>
 		</div>
 
-
 		<div class="checkboxList">
 			<label for="items">Language Exports</label>
 			<ul class="items">
-				{{range $index, $element := .Languages}}
-				<li><input type="checkbox" name="Languages[]" value={{$element.ISO6391}} /><span class="uppercase">{{$element.ISO6391}}</span> - {{$element.LanguageName}} </li>
+
+				{{$Selected := .SelectedLanguages}}
+				{{$Language := .Languages}}
+
+				{{range $index, $Language := $Language}}
+					{{$isSelected := false}}
+
+					{{range $index2, $Selected := $Selected}} 
```

Comment:
I think it's a good idea to be consistent with variable naming.

$Language should be $language.

Also try to avoid unnecessary variable definition or overwriting variables if they have different types.
Here you're assigning $Selected to $Selected while changing the type, and hiding the outer variable (:= makes a new variable).

I would just use the argument directly in the range: range $index, $language := .Languages

You also don't need to define $index if you don't use it, just range $language := .Languages should work.